r/R/dataset-scan.R

```diff
@@ -53,6 +53,29 @@
 #' query and returns an Arrow [Table].
 #' @rdname Scanner
 #' @name Scanner
+#' @examplesIf arrow_with_dataset() & arrow_with_parquet()
+#' # Set up directory for examples
+#' tf <- tempfile()
+#' dir.create(tf)
+#' on.exit(unlink(tf))
+#'
+#' data <- dplyr::group_by(mtcars, cyl)
```

Comment:
Given we don't have dplyr as a dependency, could it be worth removing the `group_by()` call here, and manually specifying the `partitioning` argument to `write_dataset()` instead?

Reply:
Good point. I've replace the call I added as well as the one in the other example I copied from.